routers/user/oauth.go

```diff
@@ -162,9 +160,6 @@ func newAccessTokenResponse(grant *models.OAuth2Grant) (*AccessTokenResponse, *A
 
 // AuthorizeOAuth manages authorize requests
 func AuthorizeOAuth(ctx *context.Context, form auth.AuthorizationForm) {
-	errs := binding.Errors{}
-	errs = form.Validate(ctx.Context, errs)
```

Comment:
I think we need keep line 166 and add check for `errs`.

Reply:
You mean, adding a cherck `if len(errs) > 0`? And what should happen in that case?

Reply:
Yes, I think form validate should not be ignored. If there are some `errs`, we can return first `err` of `errs` I think or we can join all the `errs` as a new `err`.

Reply:
I've joined and returned them now.